R/create-rd.sh

```diff
@@ -34,4 +34,5 @@ pushd "$FWDIR" > /dev/null
 . "$FWDIR/find-r.sh"
 
 # Generate Rd files if devtools is installed
-"$R_SCRIPT_PATH/Rscript" -e ' if("devtools" %in% rownames(installed.packages())) { library(devtools); devtools::document(pkg="./pkg", roclets=c("rd")) }'
+# TODO(SPARK-30737) reenable this once packages are correctly installed in Jenkins
```

Comment:
This should be less critical since the SparkR tests themselves pass. I will summarize the summary for followup actions after I make it pass and merge.